kits/quant-x-terminal/apps/.env.example

```
@@ -0,0 +1,2 @@
+REACT_APP_LAMATIC_API_KEY=your_lamatic_api_key_here
+REACT_APP_LAMATIC_PROJECT_ID=your_lamatic_project_id_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)